.circleci/real_config.yml

```diff
@@ -4721,6 +4762,243 @@ workflows:
           requires:
             - package-and-push-system-dev-ee
 
+  test-e2e-upgrade:
```

Comment:
i'm not sure i understand the value in these tests. if i understand, each e2e run runs determined $OLD_VERSION, doesn't do anything on that version, stops it, upgrades to the $HEAD, and then runs a test suite. but i don't see much of a difference between that and just testing against $HEAD to begin with; the only difference between those systems would be that one took a brief pause at some DB migration number, but even then they all the same migrations in the same order with essentially the same data. if we created some data or an experiment on $OLD_VERSION then upgraded to $HEAD and made sure the data looked right or the experiment successfully unpaused, that would cover a lot more of our previously experienced upgrade bugs.